common/include/pcl/common/impl/common.hpp

```diff
@@ -403,30 +403,29 @@ pcl::getCircumcircleRadius (const PointT &pa, const PointT &pb, const PointT &pc
 }
 
 //////////////////////////////////////////////////////////////////////////////////////////////
-template <typename PointT> inline void 
+template <typename PointT> inline void
 pcl::getMinMax (const PointT &histogram, int len, float &min_p, float &max_p)
 {
   min_p = FLT_MAX;
   max_p = -FLT_MAX;
 
   for (int i = 0; i < len; ++i)
   {
-    min_p = (histogram[i] > min_p) ? min_p : histogram[i]; 
-    max_p = (histogram[i] < max_p) ? max_p : histogram[i]; 
+    min_p = (histogram[i] > min_p) ? min_p : histogram[i];
+    max_p = (histogram[i] < max_p) ? max_p : histogram[i];
   }
 }
 
 //////////////////////////////////////////////////////////////////////////////////////////////
 template <typename PointT> inline float
-pcl::calculatePolygonArea (const pcl::PointCloud<PointT> &polygon) 
+pcl::calculatePolygonArea (const pcl::PointCloud<PointT> &polygon)
 {
   float area = 0.0f;
   int num_points = polygon.size ();
-  int j = 0;
```

Comment:
Line 430 needs to be the declaration and assignment if you're removing the declaration from here. This is basic C++.